src/Telemetry/Tracks/class-tracks.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Telemetry: Tracks class
+ *
+ * @package Parsely\Telemetry
+ * @since   3.12.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Telemetry;
+
+use WP_Error;
+
+/**
+ * This class comprises the mechanics of sending events to the Automattic Tracks
+ * system.
+ *
+ * @since 3.12.0
+ */
+class Tracks extends Telemetry_System {
+	/**
+	 * Registers the events into WordPress hooks to activate tracking.
+	 *
+	 * @since 3.12.0
+	 *
+	 * @return void
```

Comment:
Per WP documentation standards, When a function returns `void`, we don't need to add `@return void`.

Reply:
This seems addressed, so marking it as resolved.